frontend/lib/src/theme/primitives/radii.ts

```diff
@@ -20,7 +20,8 @@ export const radii = {
   default: "0.5rem",
   xl: "0.75rem",
   xxl: "1rem",
-  xxxl: "1.25rem",
+  // Chat input enforces a more rounded look:
+  chatInput: "1.25rem",
```

Comment:
Renamed this to a more semantical name since I think we don't want the theme variable to be used by any other component or adapted based on a base radius/roundedness in advanced theming.